gradio/blocks.py

```diff
@@ -388,6 +393,18 @@ def __init__(
         self.total_runs = 0
         self.inputs_as_dict = inputs_as_dict
         self.name = getattr(fn, "__name__", "fn") if fn is not None else None
+        self.spaces_auto_wrap()
+
+    def spaces_auto_wrap(self):
+        if spaces is None:
+            return
+        if not utils.is_space():
+            return
+        wrapped_fn = spaces.gradio_auto_wrap(self.fn)
+        if wrapped_fn is self.fn:
+            return
+        print(f"Your `{self.name}` event handler has been optimized for Spaces")
```

Comment:
I don't think we need to actually print this (can get noisy for Blocks with multiple functions), but instead I meant we can include some messaging in the changelog saying that "Functions in Gradio apps will automatically get optimized for Spaces, and <here's how to disable that>"

Reply:
Ok I totally missed your point then ! I'll add the line in the changelog

Reply:
Sorry for the confusion!